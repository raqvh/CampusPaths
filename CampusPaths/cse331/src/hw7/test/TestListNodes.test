# Test nodes are properly listed for a given graph

CreateGraph testgraph
AddNode testgraph this
AddNode testgraph should
AddNode testgraph be
AddNode testgraph correct

ListNodes testgraph