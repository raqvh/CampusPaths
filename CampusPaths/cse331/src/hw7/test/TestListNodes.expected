# Test nodes are properly listed for a given graph

created graph testgraph
added node this to testgraph
added node should to testgraph
added node be to testgraph
added node correct to testgraph

testgraph contains: this should correct be
